jupyterhub_moss/spawner.py

```diff
@@ -240,31 +218,29 @@ async def _get_partitions_info(self):
         out = await self.run_command(cmd)
 
         # Parse command output
-        resources_display, resources_info = self.slurm_info_resources(out)
-        dbgmsg = "Slurm partition resources displayed as available resources: %s"
-        self.log.debug(dbgmsg, resources_display)
+        resources_info = self.slurm_info_resources(out)
         self.log.debug("Slurm partition resources: %s", resources_info)
 
-        for partition in resources_info:
-            if not all(
-                counter in resources_info[partition] for counter in RESOURCES_COUNTS
-            ):
-                errmsg = "Missing required resource counter in Slurm partition: {}"
-                raise KeyError(errmsg.format(partition))
```

Comment:
Moving this test after combining the info with the config one: Only the final result matters.